ui/src/buckets/actions/index.ts

```diff
@@ -1,4 +1,4 @@
-import {Dispatch} from 'redux-thunk'
+import {Dispatch} from 'react'
```

Comment:
can you give some background in this change _por favor_? like, why is it being imported from React now?

Reply:
`redux-thunk` doesn't actually have an exported`Dispatch` type.  So, `Dispatch` was really `any`